R/register.R

```diff
@@ -136,6 +136,18 @@ get_registered_functions <- function(decorations, tag, quiet = FALSE) {
   }
 
   out <- decorations[decorations$decoration == tag, ]
+
+  if(any(!decorations$decoration %in% c("cpp11::register", "cpp11::init"))) {
```

Comment:
maybe store the result in a variable to make this easier to read?

e.g. 

```r
bad_decorations <- !decorations$decoration %in% c("cpp11::register", "cpp11::init")

if (any(bad_decorations)) { # etc..
```

Reply:
Sure!